py/desispec/workflow/schedule.py

```diff
@@ -0,0 +1,189 @@
+#
+# See top-level LICENSE.rst file for Copyright information
+#
+"""
+desispec.workflow.schedule
+==========================
+
+Tools for scheduling MPI jobs using mpi4py
+"""
+from mpi4py import MPI
+import numpy as np
+from logging import getLogger
+
+class Schedule:
+
+    def __init__(self, workfunc, **kwargs):
+        """
+        Intialize class for scheduling MPI jobs using mpi4py 
+        
+        Args:
+            workfunc: function to do each MPI job defined using 
+                      def workfunc(comm,job):
+                          where comm is an MPI communicator 
+                          and job is the index of the job 
+            
+        Keyword Args:
+            comm:       MPI communicator (default=None)
+            njobs:      number of jobs (default=2)
+            group_size: number of MPI processes per job (default=1) 
```

Comment:
Normally we could define this with
```
def __init__(self, workfunc, comm=None, njobs=2, group_size=1):
    ....
```
so that python introspection (e.g. with pydoc) can automatically see what keywords are available and what their defaults are.  Unless there is a specific need to use generic `**kwargs`, let's use this more specific form.